dashboard/db/migrate/20210715223451_create_reviewable_projects.rb

```diff
@@ -0,0 +1,21 @@
+class CreateReviewableProjects < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reviewable_projects do |t|
+      t.integer :storage_app_id, null: false
+      t.integer :user_id, null: false
+      t.integer :level_id, null: false
+      t.integer :script_id, null: false
+
+      t.timestamps
+
+      t.index [:storage_app_id, :user_id],
+        name: 'index_reviewable_projects_on_storage_app_id_and_user_id'
+    end
+
+    reversible do |dir|
```

Comment:
This I think you can skip because you're not storing any text -- I included it in the `code_review_comments` migration because we're storing free form text there and I wanted to give students the ability to include emojis in comments.